components/mailbox/src/index.html

```diff
@@ -18,6 +18,10 @@
 
         // mailbox.all_threads = [thread1]; // enable to overwrite thread_id fetching
         mailbox.actions_bar = ["selectall", "star", "delete", "unread"];
+        // mailbox.keyword_to_search = "Hi";
+        // mailbox.query_parameters = {
+        //   from: "tips@nylas.com"
+        // }
```

Comment:
✂️ ?

Reply:
This is for showing an example of initializing these props

Reply:
I appreciated this being here actually! Let's leave it here please.